feature_tests/features/product_team.failure.feature

```diff
@@ -4,30 +4,6 @@ Feature: Product Teams Failure Scenarios
     Given ODS Organisations:
       | ods_code | name    |
       | H8S7A    | BURENDO |
-    And Users:
-      | user_id          | name      |
-      | test@example.org | Test User |
     When User "test@example.org" creates Product Team {00702d39-e65f-49f5-b9ef-6570245bfe17} called " " for supplier H8S7A
     Then the operation is not successful
-    And the error is BadEntityNameError
-
-# # COMMENTED OUT SINCE CAN'T TEST THIS WITH DOMAIN CODE
-# Scenario: Failure - unknown supplier
-# Given ODS Organisations:
-# | ods_code | name |
-# And Users:
-# | user_id          | name      |
-# | test@example.org | Test User |
-# When User "test@example.org" creates Product Team {00702d39-e65f-49f5-b9ef-6570245bfe17} called "BURENDO" for supplier H8S7A
-# Then the operation is not successful
-# And the error is InvalidTypeError
-# # COMMENTED OUT SINCE CAN'T TEST THIS WITH DOMAIN CODE
-# Scenario: Failure - unknown user
-# Given ODS Organisations:
-# | ods_code | name    |
-# | H8S7A    | BURENDO |
-# And Users:
-# | user_id | name |
-# When User "test@example.org" creates Product Team {00702d39-e65f-49f5-b9ef-6570245bfe17} called "BURENDO" for supplier H8S7A
-# Then the operation is not successful
-# And the error is NotFoundError
+    And the error is ValidationError
```

Comment:
I think we need to inspect the validation error here as could be misleading and at worst will allow bugs to silently pass